src/engine.go

```diff
@@ -82,6 +83,16 @@ func (e *engine) renderText(text string) {
 	if e.activeSegment.Background != "" {
 		defaultValue = fmt.Sprintf("<%s>\u2588</>", e.activeSegment.Background)
 	}
+
+	// hyperlink matching
+	results := findNamedRegexMatch("(?P<all>(?:\\[(?P<name>.+)\\])(?:\\((?P<url>.*)\\)))", text)
```

Comment:
I would make this a function in `ansi_formats.go` so it can also be tested properly.